Formula/g/glbinding.rb

```diff
@@ -23,31 +23,40 @@ class Glbinding < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "glfw" => :test
 
   on_linux do
     depends_on "mesa"
     depends_on "mesa-glu"
   end
 
   def install
-    ENV.cxx11
-    system "cmake", ".", *std_cmake_args, "-DGLFW_LIBRARY_RELEASE="
-    system "cmake", "--build", ".", "--target", "install"
+    # Force install to use system directory structure
+    inreplace "CMakeLists.txt", "set(SYSTEM_DIR_INSTALL FALSE)", "set(SYSTEM_DIR_INSTALL TRUE)"
+
+    # NOTE: Can remove `OPTION_BUILD_OWN_KHR_HEADERS=ON` (at least on Linux)
+    # if we add `libglvnd` formula and use it as part of OpenGL solution.
+    system "cmake", "-S", ".", "-B", "build", "-DOPTION_BUILD_OWN_KHR_HEADERS=ON", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
   end
 
   test do
     (testpath/"test.cpp").write <<~EOS
       #include <glbinding/gl/gl.h>
-      #include <glbinding/Binding.h>
+      #include <glbinding/glbinding.h>
+      #include <GLFW/glfw3.h>
       int main(void)
       {
-        glbinding::Binding::initialize();
+        glbinding::initialize(glfwGetProcAddress);
```

Comment:
Wasn't sure if there was a lighter weight example. Most snippets upstream use GLFW or Qt and the original code no longer works.